### DISCUSSION TITLE
Include TOM option in Vertipaq Analyzer

### DISCUSSION THREAD

**Discussion by SharmaAntriksh** [DaxStudio/DaxStudio#1177] [General]
What's the actual use case of this option? from DS perspective I don't see any difference when opening 2 files w/wo TOM.
![image](https://github.com/DaxStudio/DaxStudio/assets/73054674/feb72db5-dbe8-4c3e-9026-6bf82a1411c5)

**Comment by marcosqlbi** [comment]
DAX Studio does not consume TOM when you read a VPAX file, but other tools can use it:
- Tabular Editor can open that TOM file if you open a VPAX file (so you have TOM and VPAX aligned when you investigate). In general, you can use it as a sort of archive of statistics and related model used, regardless of the tool.
- DAX Optimizer provides better results with a full TOM model (it also works without TOM, but you might have a less accurate evaluation for certain conditions
